app/bundle/src/main/scala/org/bitcoins/bundle/gui/NeutrinoConfigPane.scala

```diff
@@ -76,12 +97,18 @@ class NeutrinoConfigPane(
   }
 
   def getConfig: Config = {
-    val configStr =
+    var configStr =
       s"""
+         |bitcoin-s.network = ${networkComboBox.value.value}
          |bitcoin-s.node.mode = neutrino
          |bitcoin-s.node.peers = ["${peerAddressTF.text.value}"]
          |""".stripMargin
-
+    // Auto-enable proxy for .onion peers
+    if (peerAddressTF.text.value.contains(".onion")) {
```

Comment:
instead of doing a `var` and this it would be cleaner (we tend to want to avoid `var`) to do something like

```
val proxyConfStr = if (peerAddressTF.text.value.contains(".onion"))  {
...
} else ""

configStr = proxyConfStr + ...